library/Ivoz/Provider/Domain/Model/UsersCdr/UsersCdrDto.php

```diff
@@ -17,10 +17,28 @@ public static function getPropertyMap(string $context = '', string $role = null)
                 'disposition' => 'disposition',
                 'id' => 'id'
             ];
+
+            if ($role !== 'ROLE_COMPANY_USER') {
+                $response += [
+                    'userId' => 'user',
+                    'friendId' => 'friend',
+                    'extensionId' => 'extension',
+                ];
+            }
         } else {
             $response = parent::getPropertyMap(...func_get_args());
         }
 
+        if ($role === 'ROLE_BRAND_ADMIN') {
+            unset($response['brandId']);
```

Comment:
This entity is not currently exposed in Brand API (due to an error), so this condition is debatable